source/reward_api.rst

```diff
@@ -23,7 +23,7 @@ store                          Yes         Integer indicating store ID to be que
 only_redeemable_in_this_store  No          `true` or `false`. Defaults to `false`.
                                            If `true`, only rewards redeemable in given store will be returned.
 include_inactive_vouchers      No          Boolean indicating response include inactive vouchers or not
-channel                        No          Integer indicating channel number to be queried for reward. Available values are ``1`` for Mobile app, ``2`` for POS, ``3`` for kiosk, ``4`` for web, ``5`` for Android or ``6`` for iOS
+channel                        No          Integer indicating channel number to be queried for reward.
```

Comment:
boleh tolong dikasih linknya sekalian mungkin bar?
kayak disini
![image](https://user-images.githubusercontent.com/22322537/204966753-dc8f5b35-bdf7-4fe4-bfce-d30b4d5445e3.png)
biar bisa jadi link gini: https://docs.stamps.co.id/en/latest/activity_api.html#activity-type

Reply:
i see, thank you